src/models/customModel.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  Model,
+  Deployable,
+  Web3Connection,
+  Web3ConnectionOptions,
+} from '@taikai/dappkit';
+import {AbiItem} from 'web3-utils';
+
+//TODO: Move to dappKit
```

Comment:
this todo is a lie :P (like we spoke about, you made the perfect implementation :D)